### ISSUE TITLE
Install documentation outdated / incorrect?

### ISSUE THREAD

**Issue by mvexel** [maproulette/maproulette-backend#993]
Here's my environment:

```
❯ javac --version
javac 11.0.16.1
➜ scalac --version
Scala compiler version 2.13.2 -- Copyright 2002-2020, LAMP/EPFL and Lightbend, Inc.
➜ sbt --version
sbt version in this project: 1.7.1
sbt script version: 1.3.8
```

When I follow the instructions and send a test request to the API to create a project, I get:

`com.google.common.util.concurrent.UncheckedExecutionException: java.lang.IllegalStateException: Recursive load of: org.maproulette.framework.service.ServiceManager.<init>()`

Either I am doing something wrong or the instructions are out of data / incorrect.

**Comment by mvexel** [comment]
@ljdelight I would like for you to walk me through setting up the environment and application sometime.

**Comment by ljdelight** [comment]
Resolved with #998. @mvexel lmk if you'd like to go through the setup
